qaframework-bdd-tests/cypress.json

```diff
@@ -8,12 +8,12 @@
   "defaultCommandTimeout": 120000,
   "baseUrl": "https://dev3.openmrs.org/openmrs/spa",
   "env": {
-    "API_BASE_URL": "https://openmrs-spa.org/openmrs/ws/rest/v1",
-    "ADMIN_USERNAME": "admin2",
+    "API_BASE_URL": "https://dev3.openmrs.org/openmrs/ws/rest/v1",
+    "ADMIN_USERNAME": "admin",
     "ADMIN_PASSWORD": "Admin123",
-    "ADMIN_UUID": "fbd7a058-88c4-4747-b572-32aaf8ef6ac9",
-    "DEFAULT_LOCATION_UUID": "6351fcf4-e311-4a19-90f9-35667d99a8af",
-    "DEFAULT_LOCATION_NAME": "Registration Desk",
+    "ADMIN_UUID": "d9d07fcd-94f9-11ec-af68-0242ac1e0002",
+    "DEFAULT_LOCATION_UUID": "44c3efb0-2583-4c80-a79e-1f756a03c0a1",
+    "DEFAULT_LOCATION_NAME": "Outpatient Clinic",
```

Comment:
Is there any reason why changing the `ADMIN_UUID`, `DEFAULT_LOCATION` and `DEFAULT_LOCATION_NAME`?

Reply:
@kdaud the adminUuid at https://dev3.openmrs.org is different from that used at https://openmrs-spa.org/openmrs/spa

Reply:
What about `DEFAULT_LOCATION?`